rhobs/test/import/import_test.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2019 The prometheus-operator Authors
```

Comment:
Ahem .. 2023 :p And not prometheus-authors 
```suggestion
// Copyright 2023 
```

Reply:
updated